tools/nimweb.nim

```diff
@@ -263,8 +263,15 @@ proc findNim(): string =
 
 proc exec(cmd: string) =
   echo(cmd)
-  let (_, exitCode) = osproc.execCmdEx(cmd)
-  if exitCode != 0: quit("external program failed")
+  let (outp, exitCode) = osproc.execCmdEx(cmd)
+  if exitCode != 0: 
+    let
```

Comment:
Formatting is somewhat nicer with `===` lines around the output, I think. especially since `pdflatex` tends to produce a lot of output. But I can change it if you really do not like it.